onnxruntime/test/testdata/transform/fusion/reshape_fusion_gen.py

```diff
@@ -232,11 +233,9 @@ def save_model(graph, file_name):
         helper.make_node("Shape", ["unsqueeze0_out"], ["dummy_out"], "dummy"),
         
         helper.make_node("Shape", ["SubgraphRoot"], ["shape2_out"], "shape2"),
-        helper.make_node("Squeeze", ["shape2_out"], ["squeeze_out"], "squeeze"),
-        helper.make_node("Div", ["squeeze_out", "div_init"], ["div_out"], "div"),
-        helper.make_node("Unsqueeze", ["div_out"], ["unsqueeze2_out"], "unsqueeze2", axes=[0]),
+        helper.make_node("Slice", ["shape2_out", "slice_starts", "slice_ends"], ["slice_out"], "slice1"),
```

Comment:
Using Shape -> Squeeze -> Div -> Unsqueeze -> Concat was not "semantically correct". Shape would have [10, 20, 30] and squeeze and div were no-ops and unsqueeze would make the shape [1, 3] and so Concat had to basically concat 3 tensors of shape [1], [1] (from the first two branches) and [1, 3] which is incorrect. Hence, this pedantic modification of the tests.